packages/plugin-ext/src/plugin/languages/token-provider.ts

```diff
@@ -0,0 +1,26 @@
+/********************************************************************************
+ * Copyright (C) 2018 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+import { Event } from 'vscode-jsonrpc';
+
+/**
+ * Provides a token (for now it's a NONE)
+ */
+export function createToken(): any {
```

Comment:
I guess it would be useful for other modules also. So maybe move the `token-provider.ts` up one level?

Reply:
agree

Reply:
changed